framework/PostProcessors/BasicStatistics.py

```diff
@@ -544,13 +544,12 @@ def _computeWeightedPercentile(self,arrayIn,pbWeight,percent=0.5):
     sortedWeightsAndPoints = np.insert(np.asarray(list(zip(pbWeight[idxs],arrayIn[idxs]))),0,[0.0,arrayIn[idxs[0]]],axis=0)
     weightsCDF             = np.cumsum(sortedWeightsAndPoints[:,0])
     try:
-      index = utils.find_le_index(weightsCDF,percent)
+      index = utils.first(np.where(weightsCDF <= percent))[-1]
```

Comment:
The following is the document from the numpy website (the latest version). It seems to me they claim np.where return the elements instead of index
https://numpy.org/doc/1.18/reference/generated/numpy.where.html?highlight=where#numpy.where

Instead np.argwhere return the indices.
https://numpy.org/doc/1.18/reference/generated/numpy.argwhere.html?highlight=where#numpy.argwhere

Reply:
From https://numpy.org/doc/1.18/reference/generated/numpy.where.html?highlight=where#numpy.where
```
Note
When only condition is provided, this function is a shorthand for np.asarray(condition).nonzero(). Using nonzero directly should be preferred, as it behaves correctly for subclasses. The rest of this documentation covers only the case where all three arguments are provided.
```

I have tried it, and np.asarray(condition).nonzero() return the correct indices.

Reply:
addressed as yuou suggested